front/app/containers/ProjectsShowPage/shared/pb/PBExpenses.tsx

```diff
@@ -407,10 +413,31 @@ class PBExpenses extends PureComponent<Props & InjectedIntlProps, State> {
             </ProgressBar>
 
             <Footer viewMode={viewMode}>
+              {viewMode === 'column' && (
```

Comment:
Changes based on the new UI. See Figma design in the specs (link in the Conversation tab)